src/components/TopTenWrapper.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+    <div id="calculator" class="pb-20">
+        <div v-if="rankedList.length > 1"
+            class="lg:grid hidden grid-cols-3 gap-4 w-5/6 mx-auto calculator-box auto-rows-fr">
+
+            <div class="col-span-1 calculator-list">
+                <TopTenSidebar :ranked-list="rankedList" :activeItemId="activeItemId" :allowDelete="allowDelete" />
+                <button class="btn-primary mt-4" @click="download()">Download List</button>
+            </div>
+            <div class="col-span-2 h-full">
+                <div class="calculator-details">
+                    <div class="container-header">
+                        <h2>
+                            <span class=" text-ctid-light-purple">
+                                {{ rankedList[activeItemId].tid }}
+                            </span>
+                            {{ rankedList[activeItemId].name }}
+                        </h2>
+                    </div>
+                    <div>
+                        <TopTenDetails :technique="rankedList[activeItemId]" />
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <div v-if="rankedList.length > 1" class="w-5/6 mx-auto lg:hidden block">
+            <TopTenAccordion :ranked-list="rankedList" :activeItemId="activeItemId" :allowDelete="allowDelete" />
+            <button class="btn-primary mt-10" @click="download()">Download List</button>
+        </div>
+        <div v-if="rankedList.length <= 1" class="mx-auto italic w-max">
+            No techniques found for the given criteria.
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import { useCalculatorStore } from "../stores/calculator.store";
+import TopTenSidebar from "../components/TopTenSidebar.vue"
+import TopTenDetails from "../components/TopTenDetails.vue"
+import TopTenAccordion from "../components/TopTenAccordion.vue"
+import downloadjs from "downloadjs";
```

Comment:
To ensure we can perform type checking against `downloadjs`. Install the TypeScript type declarations for the library:

```bash
npm install --save-dev @types/downloadjs
```

And register the types with the application by adding `"@types/downloadjs"` to `types` in `tsconfig.app.json`:

```ts
{
  ...
  "compilerOptions": {
    ...
    "types": [
      "vite/client",
      "@types/downloadjs"
    ]
  }
}
```